articles/communication-services/how-tos/router-sdk/handle-offer.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Handle an offer in Job Router
+titleSuffix: An Azure Communication Services how-to guide
+description: Use Azure Communication Services SDKs to manage the job offer
+author: marcma
+ms.author: marcma
+ms.service: azure-communication-services
+ms.topic: how-to 
+ms.date: 01/18/2022
+ms.custom: template-how-to
+
+#Customer intent: As a developer, I want to handle job offers when coming in.
+---
+
+# Handle an offer
+
+This guide outlines the steps to observe and response a Job Router offer.
+
+[!INCLUDE [Private Preview Disclaimer](../../includes/private-preview-include-section.md)]
+
+## Prerequisites
+
+- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F). 
+- A deployed Communication Services resource. [Create a Communication Services resource](../../quickstarts/create-communication-resource.md).
+- Optional: Complete the quickstart to [get started with Job Router](../../quickstarts/router/get-started-router.md)
+
+## Query the worker to observe the job offers
+Following the quickstart, use the Job Router client connection to query the Worker and observe the ID of the Job against the ID 
+
+```csharp
```

Comment:
I don't recommend indenting fenced, blocks it does nothing and makes other indents weird. Here for example, you're newlining console.writeline which isn't needed if you dont indent the code. 

The Docs platform , doesn't care about this initial indentiation.